src/sels_fe/src/store/reducers/question.js

```diff
@@ -0,0 +1,121 @@
+import {
+  POST_QUESTION_SUCCESS,
+  POST_QUESTION_FAIL,
+  GET_QUESTION_SUCCESS,
+  GET_QUESTION_FAIL,
+  GET_QUESTION_LIST_SUCCESS,
+  GET_QUESTION_LIST_FAIL,
+  UPDATE_QUESTION_SUCCESS,
+  UPDATE_QUESTION_FAIL,
+  DELETE_QUESTION_SUCCESS,
+  DELETE_QUESTION_FAIL,
+} from "../actions/Question/action-types"
+
+export const initialState = {
+  question_list: [],
+  msg: "",
+  error: null,
+}
+
+const postQuestionSuccess = (state = initialState, action) => {
+  return {
+    ...state,
+    question_list: [...state.question_list, action.payload.question],
```

Comment:
Since data received will be bulked, update to append array of objects
```suggestion
    question_list: [...state.question_list, ...action.payload.question],
```